scraper/scrapers/__init__.py

```diff
@@ -0,0 +1,18 @@
+from urllib.parse import urlparse
+from selenium import webdriver
+
+from . import (
+    webnovel,
+)
+
+
+class NovelScraperFactory:
```

Comment:
One class per file? /scraper/NovelScraperFactory.py

Don't put anything in __init__.py unless you do a library and need to execute code at startup.